### DISCUSSION TITLE
Taskfile not on twitter?

### DISCUSSION THREAD

**Discussion by ssbarnea** [go-task/task#749] [General]
Why taskfile is almost impossible to find on social media, especially on twitter?

**Comment by andreynering** [comment]
Can you elaborate on why do you think a Twitter account would be helpful?

**Reply by ssbarnea** [reply]
Since I discovered taskfile project relatively recently, I observed that its lack of presence on youtube, twitter, reddit. I think that improving its social presence would benefit more he project and make more look deeper into it and consider it for their projects. I personally do not watch youtube tech videos so much but i know that for the younger generations that is very important, especially in the initial "discovery" stages.

Trying to find a replacement for whatever ancient and limiting build-tool is a very common question that appears online and having a twitter account might help with that. This account could be shared among few folks that want to help popularise the project. 

I hope that helps.

**Reply by ghostsquad** [reply]
This would create additional burden on the maintainers of this code, without any benefits derived from the increased burden.

**Reply by ssbarnea** [reply]
I intentionally did not use the term maintainers when I proposed it, as I am sure they would likely prefer to spend time fixing bugs and implementing new features instead of doing "PR" work. Still, I bet there are users passionate about go-task that might want to help in different ways, like answering the discussions forum, or engaging online with others (like twitter).

**Reply by andreynering** [reply]
In case you are interested, I do have some bookmarket links on browser with searches, like [this](https://nitter.net/search?f=tweets&q=taskfile.dev) and [this](https://nitter.net/search?f=tweets&q=go-task).

I don't think we have such a person interested in keeping that social media, or even much to post actually. I could create an account and take care of that, but I would post relatively unoften.

**Reply by ssbarnea** [reply]
Give me few days of playing with taskfile more, so I can decide if my team (ansible devtools) will start to use it. If that happens, i will be able to become a promoter of the tool. I have being looking for something like taskfile for two decades, ended-up trying something an year ago with https://github.com/pycontribs/mk but I realised that I am unlikely to find the time to bring it to production and searched again. Lucky I found taskfile.

Because taskfile is written in go, it is unlikely that I would be able to make many contributions, but I might try to help in other ways.
